apps/src/music/views/TimelineSimple2Events.tsx

```diff
@@ -110,24 +119,34 @@ const TimelineSimple2Events: React.FunctionComponent<
   return (
     <div id="timeline-events">
       <div id="timeline-events-function-extents">
-        {uniqueFunctionExtentsArray.map((uniqueFunction, index) => (
-          <div
-            key={index}
-            style={{
-              position: 'absolute',
-              backgroundColor:
-                index === 0 ? 'rgba(0 0 0 / 0)' : 'rgba(255 255 255 / 0.12)',
-              borderRadius: 8,
-              left: paddingOffset + (uniqueFunction.left - 1) * barWidth,
-              width: (uniqueFunction.right - uniqueFunction.left) * barWidth,
-              top: 32 + uniqueFunction.top * eventHeight,
-              height:
-                (uniqueFunction.bottom - uniqueFunction.top) * eventHeight - 3,
-            }}
-          >
-            &nbsp;
-          </div>
-        ))}
+        {uniqueFunctionExtentsArray
+          .filter(uniqueFunction => uniqueFunction)
```

Comment:
minor - perform the `filter` in the `useMemo` above where `uniqueFunctionExtentsArray` is assigned, to save on the formatting change here?